### ISSUE TITLE
Allow users to download assigned transects

### ISSUE THREAD

**Issue by CrisSevilleja** [BiologicalRecordsCentre/ABLE#524]
At the moment, a normal user can only download data submitted by him/herself in My downloads https://butterfly-monitoring.net/mydata/downloads and My Annual Reports https://butterfly-monitoring.net/annual-report. 

I would like users can download all the data to the transects they have assigned. In protected areas or other ways, several users share a transect, so they can see the visits done by others but not download data submitted by others (only download the data they submitted). This is an inconvenience because supervisors of rangers or other roles don't have access to the data of several transects they supervised. 

To sum up, please allow users to download all the data for the transects they have assigned

@JimBacon @andrewvanbreda I don't know who can do this. Thanks in advance

**Comment by andrewvanbreda** [comment]
Hi @CrisSevilleja 
Looking at code, think this can be done from a technical point of view.
@DavidRoy to confirm go ahead.

**Comment by DavidRoy** [comment]
@andrewvanbreda yes please go ahead with this

**Comment by CrisSevilleja** [comment]
Hi @DavidRoy can you approve Andrew going further on this? This is one of the points Daniela from Austria mentioned and other countries would like to have. thanks

**Comment by andrewvanbreda** [comment]
Hi @CrisSevilleja 
This one is on my list. 
However I need to work with Gary van Breda to get this working, so I plan to look at this with him in the next few weeks.
The problem is that there are some elements of these screens (such as buttons) which only appear when there is "My Data" available. This means when we change what we mean by "My Data" (so it now includes data from other people), the logic of the site also needs to be changed so the buttons and other areas are shown at the correct time.

**Comment by CrisSevilleja** [comment]
Ok @andrewvanbreda  good it is on your list to do soon. 
I understand the complication. I think regarding what is "My data", it would mean all the data of the transects assigned to an account, anyway when downloading data the name of the recorder should appear and that will be the distinction on who did the survey. But maybe, that is a way we don't want to follow and keep separate what is My data (user data) from other users' data which the transect is shared. 

thanks for going further with this.

**Comment by CrisSevilleja** [comment]
Coming back to this, as recently two coordinators from different countries asked me the same, allow users to download all the data of the assigned transects they have. 

For giving an example and check it later: user Indicia ID 461983 has the transect EBMS:Italia:186, where 14 visits were done by him this year. The rangers of that Protected Area, user ID Indicia 461956, are not able to download the 14 visits done this year. They can see the visits done in this transect perfectly but the occurrence and visit files downloaded from My downloads are empty. 

@andrewvanbreda I know you are busy, not sure how much it will take you. Thank you in advance!

**Comment by andrewvanbreda** [comment]
Hi @CrisSevilleja 
Okay, I will try to bump this up my task list.
The My Downloads won't be able to support this using the existing buttons, but I might hopefully be able to add new buttons to support this function. In the meantime can the rangers be provided with the required data using the download from the other users

**Comment by CrisSevilleja** [comment]
Thank you @andrewvanbreda 

It's only needed on the transect buttons for the assigned transects. So far 15min counts are unique for users and moth traps we didn't reach there yet. 
Yes, meanwhile I shared the data they required.
